core/src/main/kotlin/com/github/zeldigas/text2confl/core/config/ConverterConfig.kt

```diff
@@ -8,15 +8,15 @@ import io.ktor.http.*
 import java.nio.file.Path
 
 data class ConverterConfig(
-    val titlePrefix: String,
```

Comment:
formatting. Could you please try https://github.com/zeldigas/text2confl/blob/master/.ide/intellij-codestyle.xml file if you use IntelliJ.

Reply:
Done, not sure if i see any changes.